src/main/app/claims/models/response/responseCommon.ts

```diff
@@ -1,30 +1,30 @@
+import { ResponseType } from 'claims/models/response/responseType'
+
+import { YesNoOption } from 'claims/models/response/core/yesNoOption'
+
 import { PartyType } from 'common/partyType'
 import { Party } from 'claims/models/details/yours/party'
 import { Individual } from 'claims/models/details/yours/individual'
-import { Company } from 'claims/models/details/yours/company'
 import { SoleTrader } from 'claims/models/details/yours/soleTrader'
+import { Company } from 'claims/models/details/yours/company'
 import { Organisation } from 'claims/models/details/yours/organisation'
-import { StatementOfTruth } from 'claims/models/statementOfTruth'
 
-export enum ResponseType {
-  FULL_DEFENCE = 'FULL_DEFENCE'
-}
+import { StatementOfTruth } from 'claims/models/statementOfTruth'
 
 export interface ResponseCommon {
   responseType: ResponseType
-  freeMediation: string
-  moreTimeNeeded?: string
   defendant: Party
+  moreTimeNeeded?: YesNoOption
```

Comment:
why a field rather than a boolean?

Reply:
Because it is like that on the API side :(

I will aim to remove `YesNoOption` from the claim-store model but that will be in the future. At least for now I am stopping people from using `YesNoOption` in CMC domain model ;)

Reply:
could use booleans here? and not let it creep in anymore?

Reply:
I could define boolean and map it at deserialisation time but I personally do not see much benefit from doing it. With this when we remove YesNoOption from backend we would need to revisit that piece of code anyway.